Z/ZPares/build_tarballs.jl

```diff
@@ -0,0 +1,103 @@
+    cp lib/libzpares.a $prefix/lib/
+elif [[ $target == *"w64-mingw32"* ]]; then 
+    cp Makefile.inc/make.inc.gfortran.seq ./make.inc
+    make BLAS="-L$libdir -lopenblas" LAPACK="-L$libdir -lopenblas" FFLAG="-O3 $Rankmismatch -shared -fPIC"
+    gfortran -O3 -shared -fPIC zpares_wrapper.f90 -I./include -L./lib -lzpares  -L$libdir -lopenblas -o zpares_wrapper.a
+    ld -shared -o zpares.so --whole-archive zpares_wrapper.a 
+    cp lib/* $prefix/bin/
+    cp zpares.so lib/
+    cp lib/zpares.so $prefix/lib/libzpares.dll
+    cp include/zpares.mod $prefix/include/
+    cp zpares_wrapper.mod $prefix/include/
+    cp lib/libzpares.a $prefix/lib/
+else
+    cp Makefile.inc/make.inc.gfortran.seq ./make.inc
+    make BLAS="-L$libdir -lopenblas" LAPACK="-L$libdir -lopenblas" FFLAG="-O3 $Rankmismatch -shared -fPIC"
+    gfortran -O3 -shared -fPIC zpares_wrapper.f90 -I./include -L./lib -lzpares  -o zpares_wrapper.a
+    ld -shared -o zpares.so --whole-archive zpares_wrapper.a 
+    cp zpares_wrapper.a $prefix/lib/
+    cp zpares.so lib/
+    cp lib/zpares.so $prefix/lib/libzpares.so
+    cp include/zpares.mod $prefix/include/
+    cp zpares_wrapper.mod $prefix/include/
+    cp lib/libzpares.a $prefix/lib/
+fi
+"""
+
+
+#    cp -r examples $prefix/
+# These are the platforms we will build for by default, unless further
+# platforms are passed in on the command line
+platforms = [
+    Platform("x86_64", "windows"; ),
+    Platform("i686", "windows"; ),
+    Platform("x86_64", "macos"; ),
+    Platform("aarch64", "macos"; ),
+    Platform("i686", "linux"; libc = "glibc"),
+    Platform("x86_64", "linux"; libc = "glibc"),
+    Platform("aarch64", "linux"; libc = "glibc"),
+    Platform("armv7l", "linux"; call_abi = "eabihf", libc = "glibc"),
+    Platform("powerpc64le", "linux"; libc = "glibc"),
+    Platform("i686", "linux"; libc = "musl"),
+    Platform("x86_64", "linux"; libc = "musl"),
+    Platform("aarch64", "linux"; libc = "musl"),
+    Platform("armv7l", "linux"; call_abi = "eabihf", libc = "musl"),
+    Platform("x86_64", "freebsd"; )
+]
+
+platforms = expand_gfortran_versions(platforms)
+
+
+# The products that we will ensure are always built
+products = [
+    LibraryProduct("libzpares", :libzpares)]
+
+# Dependencies that must be installed before this package can be built
+dependencies = [
+    Dependency(PackageSpec(name="OpenBLAS_jll", uuid="4536629a-c528-5b80-bd46-f80d51c5b363"))
+    Dependency(PackageSpec(name="OpenBLAS32_jll", uuid="656ef2d0-ae68-5445-9ca0-591084a874a2"))
+    Dependency(PackageSpec(name="CompilerSupportLibraries_jll", uuid="e66e0078-7015-5450-92f7-15fbd957f2ae"))
+
+]
+
+# Build the tarballs, and possibly a `build.jl` as well.
+build_tarballs(ARGS, name, version, sources, script, platforms, products, dependencies; julia_compat="1.6", preferred_gcc_version = v"9.1.0")
+#build_tarballs(ARGS, name, version, sources, script, platforms, products, dependencies; julia_compat="1.6", preferred_gcc_version = v"10.2.0")
```

Comment:
Please don't use very recent versions of GCC, that hurts compatibility.  Besides, it's relatively useless when you expand the libgfortran version since you're going to build anyway with different GCC versions.
```suggestion
build_tarballs(ARGS, name, version, sources, script, platforms, products, dependencies; julia_compat="1.6")
```

Reply:
Thanks. You are right.